### PR TITLE
Sync label

### LINKED ISSUE #58: Add data-cy attribute to the elements
https://www.loom.com/share/5d55b39d84dc4ef7a01ef70a69de4307

We need to add an attribute as `data-cy` to the HTML elements. The prop name would be `dataCy`.
This would be used by Cypress to uniquely identify the elements that would be used in Automation testing.

### PR DESCRIPTION
**Pull request by yedhink**
@edwinbbu _a

Please review and see if this is good to be merged.

cc: @goutham-subramanyam 

Fixes #58 , #42 
We can pass other props and it reflects in class attrs:
Screenshot_20201204_123258

### CODE REVIEW

File: lib/components/Label.js [hunk 1]
```diff
@@ -23,9 +33,11 @@ const Label = ({
       )}
     >
       <div className="flex flex-row">
-        <label className="m-0">
+        <label className="m-0" {...otherProps}>
           {children}
-          {required && "*"}
+          {required && (
+            <i className="ri-asterisk ri-xs align-text-bottom ml-0.5"></i>
```
Comment (edwinbbu) [inline lib/components/Label.js:39]: @yedhink _a After you suggested if l also feel the icon is an overkill. Also the icon doesn't look as good as `*`.

Comment (yedhink) [reply lib/components/Label.js:39]: @edwinbbu Should we wait for @goutham-subramanyam's input or can we make the change and merge once done?

Comment (edwinbbu) [reply lib/components/Label.js:39]: @yedhink can you give ``<span className="ml-0.5">*</span>`` and see how it looks.

Comment (yedhink) [reply lib/components/Label.js:39]: @edwinbbu 
Looks fine to me:
image

Shall i persist this change?

**Comment by yedhink** [comment]
@edwinbbu _a

Have persisted the change with span over remixicon.

If this looks good, then can you merge this? Thanks

cc: @goutham-subramanyam

File: lib/components/Label.js (hunk 1 above)
Comment (edwinbbu) [reply lib/components/Label.js]: Sure.

**Comment by edwinbbu** [comment]
LGTM!

File: lib/components/Label.js (hunk 1 above)
Comment (goutham-subramanyam) [reply lib/components/Label.js]: Looks good 👍

**Pull request merged**